cdap-data-fabric-tests/src/test/java/co/cask/cdap/data2/transaction/stream/StreamAdminTest.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.data2.transaction.stream;
+
+import co.cask.cdap.api.flow.flowlet.StreamEvent;
+import co.cask.cdap.data.file.FileWriter;
+import co.cask.cdap.data.stream.StreamFileWriterFactory;
+import co.cask.cdap.proto.Id;
+import com.google.common.base.Charsets;
+import com.google.common.collect.Lists;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ *
+ */
+public abstract class StreamAdminTest {
```

Comment:
Also maybe add a test to write some events to a stream in a namespace and verify that they do not show up in a stream in the other namespace.